app/controllers/form_templates_controller.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class FormTemplatesController < ApplicationController
+  before_action :ensure_form_templates_enabled
+
+  def index
+    form_templates = FormTemplate.all
+    render_serialized(form_templates, FormTemplateSerializer, root: "form_templates")
+  end
+
+  def show
+    params.require(:id)
+    template = FormTemplate.find(params[:id])
+    render_serialized(template, FormTemplateSerializer, root: "form_template")
+  end
+
+  private
+
+  def ensure_form_templates_enabled
+    raise Discourse::InvalidAccess.new unless SiteSetting.experimental_form_templates
+  end
+end
```

Comment:
We should add a `requests/` spec for this controller, to make sure that it works as expected for various cases: 

- when the site setting is disabled
- when an anonymous user makes the request (this should be for logged-in users only, right?) 
- when there are templates